pages/2018/abstracts.rst

```diff
@@ -0,0 +1,96 @@
+By design, Corrfunc is highly optimized and can compute \wprp for \mathcal{O}(1 million) galaxies in ~ 6 seconds on a post-2011 CPU, which is at least a factor of few faster than existing public correlation function routines.
+Corrfunc is covered by a suite of tests, extensive documentation and is publicly available at https://github.com/manodeep/Corrfunc.
+
+Data Analysis tools for JWST - Pey Lian Lim
+###########################################
+
+This talk will provide an overview of the tools being developed for JWST users.
+This includes interactive tools for data visualization as well as tools to facilitate scripted workflows for photometric and spectroscopic data analysis.
+The talk will also outline our open-source development model and highlight areas where community contributions will be most effective.
+
+The role of Python in the first detection of Gravitational Waves - Bas Swinkels
+###############################################################################
+
+The last 2 years saw the spectacular birth of Gravitational Wave astronomy, which featured the first ever detection of signals generated by colliding black holes, and a neutron star merger of which the afterglow has been observed across the EM spectrum.
+I will give a quick overview of the recent discoveries, and then explain how Python plays a key role in this field.
+I will focus on the automation of the Virgo interferometer and on analysis tools used for understanding the disturbances that are limiting the sensitivity of these complex instruments.
+
+DKIST - Stuart Mumford
+######################
+
+The Daniel K. Inouye Solar Telescope (DKIST) is a 4m solar telescope currently under construction on Haleakalā in Hawaii.
+In this talk I will give an overview of the software under development to enable users to perform scientific analysis on the data collected by the DKIST.
+This software is being developed in Python using SunPy and Astropy to provide the core tools.
+The DKIST data centre is expected to generate approximately 3 Pb of calibrated science quality data a year once telescope operations are at full capacity.
+The data centre will provide no pre-processing of the data before download, which is an extra challenge to the user software.
+The tools that will be written for these data will include, search and retrieval from the data centre, cataloguing and searching data local to the user and then reading of this data and metadata in a way which is compatible with existing tooling.
+This talk will describe these stages and the new and existing tools used in implementing them, as well as the challenges faced while doing so.
+
+Collaborations in the Extreme: the rise of open code development in the scientific community - Kelle Cruz
+#########################################################################################################
```

Comment:
See this page for abstract: https://www.simonsfoundation.org/event/collaborations-in-the-extreme-the-rise-of-open-code-development-in-the-scientific-community/